src/askui/agent.py

```diff
@@ -75,6 +75,13 @@ def click(self, instruction: str, button: Literal['left', 'middle', 'right'] = '
         self.client.mouse(x, y) # type: ignore
         self.client.click(button, repeat) # type: ignore
 
+
+    def mouse_wheel(self, direction_x: int, direction_y: int) -> None:
```

Comment:
Could we shorten this to

```suggestion
    def mouse_wheel(self, x: int, y: int) -> None:
```

without loosing meaning and add a docstring for explaining the meaning of x and y?